src/components/layout/dialogs/scanner/ScanOnDemand.jsx

```diff
@@ -69,6 +72,12 @@ export default function ScanOnDemand({
     if (scanMode === 'sendCoords') {
       demandScan()
       setScanMode('loading')
+      const timer = mode === 'scanNext' ? scanNextCooldown : scanZoneCooldown
+      useStatic.setState({
+        scannerCooldown:
+          (typeof timer === 'number' ? timer : 0) *
+          (mode === 'scanNext' ? scanCoords.length : scanZoneSize),
```

Comment:
I think `scanCoords.length` for both types would be better. It would be easier to manage cooldown especially when scanzone size 1 = 7 coords, size 2 = 19, size 3 = 37 and so on